site/_posts/2020-09-21-troubleshooting-containerized-oidc.md

```diff
@@ -255,6 +256,55 @@ The following example shows how to check for the user's request headers from the
   ... "message":"MIQ(Authenticator::Httpd#log_auth_debug)   X-REMOTE-USER-GROUPS     = \"EvmGroup-super_administrator\""}
   ```
 
+## Adjusting Log Level
+---------------------------------------------------------------------
+
+To gain additional insight to authentication issues adjust the log level to **debug** of the httpd pod,
+which can be done with the following commands:
+
+  + Login
+
+  ```bash
+  $ oc login -u admin -p <pw> https://<my-container-host>/
+  ```
+
+  + Edit the httpd configmap:
+
+  ```bash
+  $ oc edit configmaps/httpd-configs -n joev-miq
+  ``` 
+
+  + This command will open an editor where you can adjust the log level:
+
+  Search for **LogLevel** and change it from: **LogLevel warn** To: **LogLevel debug**, 
+  and save the changes.
+
+  + Get the name of the httpd pod
+
+  ```bash
+  $ oc get pods -n my-miq-project | grep httpd
+    httpd-abcdef1234-12345               1/1     Running   0          22d
+  ```
+
+  + Delete this httpd pod, which will result in a new one being created.
+
+  ```bash
+  $ oc delete pod httpd-abcdef1234-12345
```

Comment:
oc delete, oc logs, etc should all have -n my-miq-project unless you prepend these instructions with an oc project my-miq-project, then you can remove all the -n ...

Reply:
Good catch! Thank you sir!